incubator/model-core.spec/src/main/scripts/io/aklivity/zilla/specs/model/core/config/float.model.yaml

```
@@ -0,0 +1,31 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+bindings:
+  test:
+    kind: server
+    type: test
+    options:
+      value:
+        model: float
+        format: text
+        max: 1234.23
+        min: -1234.94
+        exclusiveMax: true
+        exclusiveMin: true
```

```suggestion
        range: (-1234.94,1234.23)
```
Do you think using `range` with standard `[` (inclusive) vs `(` (exclusive) would be easier to read than 4 separate properties?